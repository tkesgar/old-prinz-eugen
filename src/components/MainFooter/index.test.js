import { mount } from '@vue/test-utils'
import localVue from '../../utils/test/local-vue'
import IndexComponent from '.'

describe('MainFooter', () => {
  const wrapper = mount(IndexComponent, { localVue })

  test('should mount successfully', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })

  test('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
