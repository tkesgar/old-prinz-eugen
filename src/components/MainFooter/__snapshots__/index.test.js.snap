// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainFooter should match snapshot 1`] = `
<div
  class="footer py-4"
>
  <a
    href="https://github.com/tkesgar/prinz-eugen"
    target="_blank"
  >
    prinz-eugen
  </a>
  #
  <a
    href="https://github.com/tkesgar/prinz-eugen/commit/fce894f"
    target="_blank"
  >
    fce894f
  </a>
</div>
`;
